### DISCUSSION TITLE
How to remove this floating command line?

### DISCUSSION THREAD

**Discussion by HermesSantos** [LazyVim/LazyVim#2484] [Q&A]
![Screenshot_2](https://github.com/LazyVim/LazyVim/assets/94411142/e0f729b3-3821-411c-a16d-e5f382923040)

**Comment by dpetka2001** [comment]
#2312 should be of help. Or you can disable `noice.nvim` plugin all together, if you don't mind missing on the other features it provides and have a more classic Neovim interface.

**Reply by HermesSantos** [reply]
actually that helped, thank you
